spring-cloud-alibaba-examples/integrated-example/docs/zh/local-deployment-zh.md

```diff
@@ -20,9 +20,22 @@
 - [RocketMQ: 4.9.4 版本](https://github.com/apache/rocketmq/releases)
 - MySQL: 5.7 版本
 
+### host配置
+
+为了保证代码可以正常启动，请先配置好本机的 host 映射，在配置文件中新增如下的映射。
+```sh
+# for integrated-example
+127.0.0.1 integrated-mysql
```

Comment:
Is the quantity complete? Don't need to add all NodePort and ClusterIP of K8s as host here?